### ISSUE TITLE
Add support for Conditionals and Comparisons '>, >=, <=, <'

### ISSUE THREAD

**Issue by kenjones-cisco** [mikefarah/yq#96]
Add support for [Conditionals and Comparisons '>, >=, <=, <'](https://stedolan.github.io/jq/manual/#example75)

```
The comparison operators >, >=, <=, < return whether their left argument is greater than, greater than or equal to, less than or equal to or less than their right argument (respectively).

The ordering is the same as that described for sort, above.
```

**Comment by languitar** [comment]
Why is this closed? Is this implemented?

**Comment by vcavallo** [comment]
Am I crazy or does yq not do comparisons?

**Comment by mikefarah** [comment]
Oops - sorry I closed a bunch of issues back then that didn't seem to have anyone interested in them (then didn't see the comments). Re opening.

**Comment by mikefarah** [comment]
Fixed in 4.24.2! Sorry for the big delay!

**Comment by mikefarah** [comment]
Docs: https://mikefarah.gitbook.io/yq/operators/compare
